README.md

```diff
@@ -1,8 +1,13 @@
 # Learning about Drasi
-This repo contains code for sample applications that use Drasi as an end-to-end solution for change detection and real-time reactions. The applications will help you understand how to implement and deploy Drasi for different scenarios. Each app folder  within this repo has a README.md which details what the scenario for the app, code for the apps, requisite test data, and how to run the samples.
+This repo contains code for a tutorial and a few sample applications that use Drasi as an end-to-end solution for change detection and real-time reactions. The tutorial The applications will help you understand how to implement and deploy Drasi for different scenarios. Each app folder within this repo has a README.md which details what the scenario for the app, code for the apps, requisite test data, and how to run the samples.
+
+## Getting Started with Drasi
+#### Detect and react to your first database change using Drasi
+[This step-by-step tutorial](https://drasi.io/getting-started/) will help you get Drasi up and running quickly and show you how easy it is to create Sources, Continuous Queries, and Reactions.
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/drasi-project/learning)
```

Comment:
@agentofreality Here is a link to launch codespace that we can include in various places including drasi.io.